deliveries/milestone-delivery-template.md

```diff
@@ -1,14 +1,18 @@
 # Milestone Delivery :mailbox:
 
-> ⚡ Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones. 
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones. 
 > 
 > Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
 
 **The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
 
-* **Application Document:** Please, provide a link to the merged contract (the `.md` file in the [applications](https://github.com/w3f/Grants-Program/tree/master/applications) directory). 
-* **Milestone Number:** The number of the milestone
+* **Application Document:** Please, provide a link to the merged contract (the `.md` file in the [applications](https://github.com/w3f/Grants-Program/tree/master/applications) directory).
+* **Milestone Number:** e.g. 1
 
+**Context**
+> Please provide a short paragraph or two connecting the deliverables in this milestone and describing their purpose.
```

Comment:
Why do you need this or why do you think it’s helpful? I think in most cases this should be clarified in the application.

Reply:
I've found it not always is, plus teams are likely to have a different perspective/be able to explain it better after implementing it. I also noticed people tend to do it naturally for email deliveries where they don't have a template that tells them to just fill out a table.

Reply:
maybe make it optional? see above

Reply:
Fine by me